src/main/resources/templates/db_scripts/schema.json

```
@@ -470,6 +470,10 @@
           "fieldName": "allTitles",
           "multiFieldNames": "title,indexTitle,alternativeTitles[*].alternativeTitle,series"
         },
+        {
+          "fieldName": "contributorsName",
```

```suggestion
          "fieldName": "contributorsNames",
```
It should be plural because it concatenates the names of multiple contributors, similar to allTitles.